privacy_policy/en/privacy-notice-website.md

```diff
@@ -30,7 +30,7 @@ We only use analytic tools in order to understand the Website's traffic, and how
 
 We currently use two website analytic tools. These tools help us understand the use of our site by our visitors by collecting information. 
 
-We consider [Matomo](https://matomo.org) (matomo.adguard.com) the best option from the privacy point of view as data is not shared with third parties (service providers) and we manage it ourselves. Eventually we will switch over to using only this service.  
+We consider [Plausible](https://plausible.io) (p.adguard.com) the best option from the privacy point of view as data is not shared with third parties (service providers) and we manage it ourselves. Eventually we will switch over to using only this service.  
```

Comment:
1. Plausible does not use cookies at all.
2. It's actually `stat.adguard.com`
3. We should stress that we use a self-hosted version of Plausible. No data is shared with plausible.io or anyone else.